app/forms.py

```diff
@@ -67,3 +71,102 @@ def validateAndUpateBook(form):
+        self.fields['password2'].widget = forms.PasswordInput(attrs={'class': 'form-control', 'placeholder': 'Passwort bestätigen'})
+
+
+    class Meta(UserCreationForm.Meta):
+        model = User
+        fields = ('username','first_name', 'last_name', 'email','paypal', 'password1', 'password1')
+        widgets = {
+            'username': forms.TextInput(attrs={'class': 'form-control', 'placeholder': 'Benutzername'}),
+            'first_name': forms.TextInput(attrs={'class': 'form-control', 'placeholder': 'Vorname'}),
+            'last_name': forms.TextInput(attrs={'class': 'form-control', 'placeholder': 'Nachname'}),
+            'email': forms.EmailInput(attrs={'class': 'form-control', 'placeholder': 'E-Mail'}),
+            'paypal': forms.EmailInput(attrs={'class': 'form-control', 'placeholder': 'Paypal'}),
+            }
+
+    def save(self, commit=True):
+        user = super(RegistrationForm, self).save(commit=False)
+        user.email = self.cleaned_data['email']
+        user.paypal = self.cleaned_data['paypal']
+
+        if commit:
+            user.save()
+
+
+
+class EditProfileForm(UserCreationForm):
+
+    email = forms.EmailInput()
+    first_name = forms.TextInput()
+    last_name = forms.TextInput()
+    paypal = forms.EmailInput()
+
+    def __init__(self, *args, **kwargs):
+        super(EditProfileForm, self).__init__(*args, **kwargs)
+        self.fields['username'].help_text = None
+        self.fields['password2'].help_text = None
+        self.fields['password1'].widget = forms.PasswordInput(attrs={'class': 'form-control', 'placeholder': 'Passwort'})
+        self.fields['password2'].widget = forms.PasswordInput(attrs={'class': 'form-control', 'placeholder': 'Passwort bestätigen'})
+
+    class Meta(UserCreationForm.Meta):
+        model = User
+        fields = ('username','first_name', 'last_name', 'email','paypal', 'password1', 'password1')
+        widgets = {
+            'username': forms.TextInput(attrs={'class': 'form-control', 'placeholder': 'Benutzername'}),
+            'first_name': forms.TextInput(attrs={'class': 'form-control', 'placeholder': 'Vorname'}),
+            'last_name': forms.TextInput(attrs={'class': 'form-control', 'placeholder': 'Nachname'}),
+            'email': forms.EmailInput(attrs={'class': 'form-control', 'placeholder': 'E-Mail'}),
+            'paypal': forms.EmailInput(attrs={'class': 'form-control', 'placeholder': 'Paypal'}),
+
+            }
+
+    def save(self, commit=True):
+        user = super(EditProfileForm, self).save(commit=False)
+        user.email = self.cleaned_data['email']
+        user.email  =self.cleaned_data['paypal']
+
+        if commit:
+            user.save()
+
+class UserEditForm(ModelForm):
+    class Meta:
+        model = User
+        fields = ('username','first_name', 'last_name', 'email','paypal')
+        widgets = {
+            'username': forms.TextInput(attrs={'class': 'form-control', 'placeholder': 'Benutzername'}),
```

Comment:
Die `TextInput` sind überflüssig, da Django diese implizit nutzt, da es in der DB Strings sind. Also kannst du `username,first_name,last_name` entfernen. Placeholder brauchen wir bei dem Material-Design nicht. Die Labelbezeichnungen kannst du mit `labels` angeben:

```
labels = {
            'name': _('Writer'),
        }
```